lib/auth/webauthncli/api.go

```diff
@@ -189,3 +189,11 @@ func Register(
 	}
 	return U2FRegister(ctx, origin, cc)
 }
+
+// PreviewIfPlatformPasswordlessRegistered checks without user interaction
+// if there is any registered platform passwordless credential.
+// User can be empty, it will return credentials for all users.
+// Right now is supports only touchid.
+func PreviewIfPlatformPasswordlessRegistered(ctx context.Context, rpid, user string) bool {
```

Comment:
I am open for better naming suggestions 😉

Reply:
> I'd go one step further and say that I'd be OK calling the touchid method directly. The extra layer is not buying us anything at the moment.

I have started implementation with that approach but we will need to expose package variable from touchid package to allow overwriting it in tests. I think approach with hiding clients behind webauthncli makes some sense and it's nicer to test outside. Moreover if we integrate with windows webauthn list credentials endpoint, we will use that extra layer.

What do you thing @codingllama ?

Reply:
> I have started implementation with that approach but we will need to expose package variable from touchid package to allow overwriting it in tests

Not necessarily, you can use a var in lib/client, same as you did before.

> I think approach with hiding clients behind webauthncli makes some sense and it's nicer to test outside. Moreover if we integrate with windows webauthn list credentials endpoint, we will use that extra layer.

I'd be happy to refactor if/when we have list credential support, but at this moment I'm not convinced this would ever be the case, so I would cut the extra layer.